packages/neuron-wallet/src/env.ts

```diff
@@ -19,6 +17,11 @@ const app = electronApp ?? (remote?.app) ?? {
 const isTestMode = process.env.NODE_ENV === 'test'
 const isDevMode = !app.isPackaged && !isTestMode
 
+if (isDevMode) {
+  dotenv.config({ path: path.resolve(__dirname, '..', '.env.development.local') })
```

Comment:
Maybe `.env.local` is expressive enough?

Reply:
I prefer to keep the keyword which is more clear.

`.env.local` and `.env.development.local` are both listed in the `.gitignore` so it might be considered by someone.